decidim-elections/spec/events/decidim/elections/trustees/notify_trustee_new_election_event_spec.rb

```diff
@@ -8,27 +8,11 @@
   let(:event_name) { "decidim.events.elections.trustees.new_election" }
   let(:resource) { create(:election) }
   let(:resource_title) { resource.title["en"] }
+  let(:email_intro) { "You got added as a trustee for the #{resource_title} election." }
```

Comment:
Mind here you have a double . at the end. please make sure you fix that as well 
```suggestion
  let(:email_subject) { "You are a trustee for the #{resource_title} election.." }
  let(:email_intro) { "You got added as a trustee for the #{resource_title} election." }
```